pkgs/backy/default.nix

```diff
@@ -1,19 +1,25 @@
 {
   fetchFromGitHub,
-  fetchPypi,
+  lib,
+  stdenv,
   poetry2nix,
   lzo,
-  python310,
+  python312,
   mkShellNoCC,
   poetry,
   runCommand,
+  libiconv,
+  darwin,
+  rustPlatform,
+
```

Comment:
these are unused

Reply:
No, these are passed through as-is to https://github.com/flyingcircusio/backy/blob/9fa1a828495e14d3eb65000935022aef8083db9a/lib.nix . It's a bit crude as a construct, but I keep using the current way of backy packaging for now.

Reply:
👻